jenkins/build-trigger.jpl

```diff
@@ -36,7 +36,7 @@ GIT_DESCRIBE_VERBOSE
   Verbose output of 'git describe' at the revision of the snapshot
 COMMIT_ID
   Git commit SHA1 at the revision of the snapshot
-ARCH_LIST (x86 arm64 arm mips arc)
+ARCH_LIST (x86 arm64 arm mips arc riscv64)
```

Comment:
I guess the commit summary would need to be adjusted accordingly before merging this.

Reply:
Yup, updated.